test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4235,8 +4235,7 @@ class Comm(PBSService):
     logger = logging.getLogger(__name__)
     logprefix = 'pbs_comm: '
 
-    conf_to_cmd_map = {'PBS_COMM_NAME': '-n', 'PBS_COMM_ROUTERS': '-r',
-                       'PBS_COMM_THREADS': '-t'}
+    conf_to_cmd_map = {'PBS_COMM_ROUTERS': '-r', 'PBS_COMM_THREADS': '-t'}
```

Comment:
I looked up the command line options to pbs_comm and there is a -N option to run in standalone mode.  Is this something we need to care about?